tests/CryptoTest.php

```diff
@@ -88,14 +88,14 @@ public function testGetDefaultParams() {
     }
 
     public function testNonAESEncryptionSupport() {
-        $blowfishParams = Crypto::getDefaultParams( [
+        $cipherParams = Crypto::getDefaultParams( [
             'key' => Crypto::generateRandomKey(128),
-            'algorithm' => 'bf',
+            'algorithm' => 'sm4',
```

Comment:
Ohkay, so this change is just related to tests 👍 . LGTM : ) 
Can you add description to the PR why `bf` also is failing.